vulcanai/run.py

```diff
@@ -0,0 +1,47 @@
+from torch.utils.data import DataLoader, Subset
```

Comment:
This looks like you made an example script for the sensitivity analysis, awesome! I think it is better suited to be in the `vulcanai/examples` folder with a more informative name, thanks!

Reply:
I actually didn't mean to commit this file. It was just for debugging purposes since I couldn't interactively debug. Should I still add it in vulcanai/examples?

Reply:
yes if you can clean it up it's worth it

Reply:
Ok